### ISSUE TITLE
AlwaysShadow makes Shadowplay turn off and on again.

### ISSUE THREAD

**Issue by rhipex** [Verpous/AlwaysShadow#6]
When AlwaysShadow is not running, I can turn on Shadowplay, and it is running without issues. But the moment I turn AlwaysShadow on, it starts cycling off on off on endlessly. Might have to do something with Geforce Experience update. This started recently.

**Comment by Verpous** [comment]
I've installed the update and at first impression AlwaysShadow is working fine. If it stops working then I'll be able to investigate. For now, if you haven't already tried closing and relaunching AlwaysShadow then please do that. If it still doesn't work, maybe you can help me identify the problem by going to GeForce Experience settings -> Highlights and see what path it says temporary files are stored in. Go to that path **while Shadowplay is running** and see if there's a folder there called "9343b833-e7af-42ea-8a61-31bc41eefe2b" which contains files named "Sha<something>.tmp".

If those files aren't there, that's why AlwaysShadow thinks Shadowplay isn't running. They should be there though unless Nvidia changed something.

**Comment by HotshotGT** [comment]
I'm getting the same behavior on a fresh install of GFE. I turned on instant replay and checked the highlights folder, but there's no "9343b833-e7af-42ea-8a61-31bc41eefe2b" subfolder or "Sha.tmp" file in either location. I changed the highlights and temp folders to a custom location on my other drive, but the folder/file were not created when re-enabling instant replay.

Maybe there's another way to determine whether or not instant replay is enabled? A registry key perhaps?
